containers/concent-api/build-dependencies.txt

```diff
@@ -0,0 +1,19 @@
+gcc
+libssl-dev
+git
+pkg-config
+libffi-dev
+software-properties-common
+curl
+make
+build-essential
+libssl-dev
+zlib1g-dev
+libbz2-dev
+libreadline-dev
+libsqlite3-dev
+llvm
+libncurses5-dev
+libncursesw5-dev
+xz-utils
+tk-dev
```

Comment:
It would be better to share dependency lists between containers. Let's talk about it.